File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// PassantenfrequenzenZuerich SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class PassantenfrequenzenZuerichMakeContext
{
    public static function call(array $ctxmap, ?PassantenfrequenzenZuerichContext $basectx): PassantenfrequenzenZuerichContext
    {
        return new PassantenfrequenzenZuerichContext($ctxmap, $basectx);
    }
}

File: go/utility/make_context.go
package utility

import "github.com/voxgig-sdk/passantenfrequenzen-zuerich-sdk/core"

func makeContextUtil(ctxmap map[string]any, basectx *core.Context) *core.Context {
	return core.NewContext(ctxmap, basectx)
}
